lib/bootloader_spvm.pm

```diff
@@ -24,6 +29,14 @@ our @EXPORT = qw(
   boot_spvm
 );
 
+=head2 get_into_net_boot
+
+ get_into_net_boot();
+
+Get into SMS menu for booting from net:
+Select Boot Options -> Install/Boot Device -> Network -> Bootp -> Primary disk -> Normal Mode Boot
```

Comment:
I think this is too verbose. The code is already clarifying this. This description would be enough with **what** is done and **why** is done. Maybe just specifying anything spvm specific that needs to be known in advance to understand the code.